pkg/apis/core/validation/validation.go

```diff
@@ -5939,8 +5939,8 @@ func ValidateSecurityContext(sc *core.SecurityContext, fldPath *field.Path) fiel
 
 		if sc.Capabilities != nil {
 			for _, cap := range sc.Capabilities.Add {
-				if string(cap) == "CAP_SYS_ADMIN" {
-					allErrs = append(allErrs, field.Invalid(fldPath, sc, "cannot set `allowPrivilegeEscalation` to false and `capabilities.Add` CAP_SYS_ADMIN"))
+				if string(cap) == "SYS_ADMIN" {
+					allErrs = append(allErrs, field.Invalid(fldPath, sc, "cannot set `allowPrivilegeEscalation` to false and `capabilities.Add` SYS_ADMIN"))
```

Comment:
Even if this change is semantically correct (which I do not know if it is), it is not allowed as it is backwards incompatible (has the potential to make existing stored data invalid).

Reply:
Are you warring about log message? Ok, I think, I think if we change it here, it could break some log analyzer.